codespell_lib/tests/test_basic.py

```diff
@@ -297,15 +309,67 @@ def test_ignore(tmpdir, capsys):
     assert cs.main(d) == 1
     assert cs.main('--skip=bad*', d) == 0
     assert cs.main('--skip=bad.txt', d) == 0
-    subdir = op.join(d, 'ignoredir')
+    ignoredir = op.join(d, 'ignoredir')
+    os.mkdir(ignoredir)
+    subdir = op.join(ignoredir, 'subdir')
     os.mkdir(subdir)
     with open(op.join(subdir, 'bad.txt'), 'w') as f:
         f.write('abandonned')
     assert cs.main(d) == 2
     assert cs.main('--skip=bad*', d) == 0
-    assert cs.main('--skip=*ignoredir*', d) == 1
+    assert cs.main('--skip=*ad*', d) == 0
```

Comment:
Sort of the same with this, if it's actually gone wrong and just matched `*`, we'd be none the wiser.

Perhaps bad and badder 😆 .